assets/js/googlesitekit/widgets/default-areas.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Widgets API default areas
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const AREA_ALL_TRAFFIC = 'AllTraffic';
+export const AREA_SEARCH_FUNNEL = 'SearchFunnel';
+export const AREA_POPULARITY = 'Popularity';
+export const AREA_SPEED = 'Speed';
+export const AREA_EARNINGS = 'Earnings';
+
+export default {
+	AREA_ALL_TRAFFIC,
+	AREA_SEARCH_FUNNEL,
+	AREA_POPULARITY,
+	AREA_SPEED,
+	AREA_EARNINGS,
```

Comment:
See my other comment reply, these should be fully qualified including the context-specific prefix, so there should be 10 areas. The names should use camel-case (e.g. `dashboardAllTraffic`).

Reply:
Updated.